VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+* **UI warning:** Users will be warned by the wallet’s UI if they want to delegate to a validator that already has a significant amount of staking power
+
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+It is highly probable that they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block provisions and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called `photons`.
+
+
+## Technical requirements 
+
+### What are hardware requirements?
+
+Validators should expect to provision one or more data center locations with redundant power, networking, firewalls, HSMs and servers.
+We expect a modest level of hardware specifications will be needed initially and requirements to rise as network use rises. Participating in the testnet is the best way to learn more.
+
+### What are software requirements?
+
+In addition to running a cosmos node, validators should develop monitoring, alerting and management solutions.
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating Denial of Service attacks and syncing new nodes on the network.
+Ultimately, as the network becomes more heavily used, multigigabyte per day bandwidth is very realistic.
+
+### What does running a validator imply in terms of logistics? 
+
+A successful validator operation will require the efforts of multiple highly skilled individuals and continuous operational attention. This will be considerably more involved than running a bitcoin miner for instance.
+
+### How to handle key management?
+
+Validators should expect to run an HSM that supports ed25519 keys. Here are the current options:
+* YubiHSM 2
+* Ledger Nano S
+* Potential options
+* Ledger BOLOS SGX enclave
+* Thales nCypher support
+* Tendermint SGX enclave
+
+### What can validators expect in terms of operations? 
+
+Running effective operation is the key to avoiding unexpectedly unbonding or being slashed. This includes being able to respond to attacks, outages etc. Maintain security and isolation in your data center etc.
+
+### What are the maintenance requirements?
+
+Validators should expect to perform regular software updates to accommodate upgrades and bug fixes. There will inevitably be issues with the network early in its bootstrapping phase that will require substantial vigilance 
+
+### How can validators protect themselves from denial-of-service attacks?
```

Comment:
Let's user use DDos or denial-of-service attacks throughout.

Reply:
ok